media/css/m24/flag.scss

```
@@ -20,17 +20,83 @@
 
 .m24-c-flag-subtitle {
     font-size: $text-title-md;
-    text-wrap: balance;
-    text-wrap-style: balance;
     margin-bottom: $spacer-xl;
 }
 
+.m24-c-flag-cta {
+    margin-bottom: 0;
+}
+
 .m24-c-flag-media {
-    display: block;
-    margin-bottom: 16px;
+    --delay: 1s; // leave some space to read before introducing movement
+    --wave-h: 8px;
 
-    img {
+    svg {
+        transform: translateZ(0);
         width: 100px;
+        height: auto;
+        fill: $m24-color-green; // zilla green
+    }
+}
+
+.m24-c-flag-button {
+    $font-size: 0.75;
+    background-color: $m24-color-dark-mid-gray;
+    border: none;
+    box-shadow: none;
+    color: $m24-color-white;
+    font-family: $primary-font;
+    font-size: calc(#{$font-size} * 1rem);
+    font-weight: 600;
+    line-height: $font-size;
+    padding: 8px;
+    cursor: pointer;
+    transition: background-color $fast;
+
+    &:hover,
+    &:focus {
+        background-color: $m24-color-black; // inverts to white in dark theme section
+    }
+
+    &:active {
+        background-color: $m24-color-dark-mid-gray;
+    }
+}
+
+.m24-c-flag-button-pause,
+.m24-c-flag-button-play {
+    align-items: center;
+    display: flex;
+    gap: 8px;
+}
+
+.m24-c-flag-button-pause {
+    .m24-c-flag-button-text {
+        position: relative;
+        top: 0.0175rem; // flex alignment isn't totally centered, so we're manually adjusting
+    }
+}
+
+// less than $screen-lg
+@media screen and (max-width: 1023px) {
```

If you don't feel like hard-coding the value, there's the possibility of:

```suggestion
@media screen and (max-width: #{$screen-lg - 1}) {
```

that have been used around here somewhere already. Or was it `#{$screen-lg - 1px}`? I'm not sure about the units exactly [TBH](https://sass-lang.com/documentation/values/numbers/#units).

---

Looks like it's been used both with and without units. Which is fine because the Sass variable has units.